opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/schema-test.ts

```diff
@@ -0,0 +1,73 @@
+import gql from "graphql-tag";
```

Comment:
some tests already exist in filterKeysSchema-test.ts

Reply:
I had no idea.
Let me move it to the right place.